packages/infra/lib/ihe-stack.ts

```diff
@@ -86,6 +86,55 @@ export class IHEStack extends Stack {
       },
     });
 
+    // Create lambdas
+    const iti38Lambda = createLambda({
```

Comment:
Please use domain/business names throughout the code, unless specifically writing a ITI38 connector/interface/conversor; here and througout.